### ISSUE TITLE
OMG... new items are blank boxes in tbc....

### ISSUE THREAD

**Issue by nwardoh** [arkayenro/arkinventory#1489]
I have a raid in just a minute and new items are showing as a blank box with a flash over them and no tooltip... FIX THIS!

**Comment by IKnewOne** [comment]
It's best to add screenshots when doing these kind of issues. And don't be so demanding to the author with the last part -.-
The most recent several versions had some bugs, it's a work in progress rn. use .53 or .52 for now

**Comment by Feirlane** [comment]
Complaining and crying for a fix doesn't help anyone. I believe your issue is the same that has already been reported in #1485, check that one out and add any information on behaviour you're experiencing that has not been reported yet.

**Comment by arkayenro** [comment]
rool back to a version that is stable for you

i know the issue exists, but i cannot reproduce it so its a giant pain - i really wish i did have the issue

i also have a job, thankfully, that is getting super busy at the moment so im getting squeezed for time

**Comment by nwardoh** [comment]
I had to unassign “new item” from a block and it started working. With it
assigned it breaks.

**Comment by arkayenro** [comment]
thanks, ill take a look and see if i can duplicate that.  i dont have new item enabled

**Comment by nwardoh** [comment]
Thank you. It happened taking stuff out of the mailbox and when I picked up
the loot to lc out after the raid. I reloaded and it was still a blank
box(not in the assigned window, it was in the default block). I removed new
item from the assignments and reloaded and it worked like expected.

**Comment by arkayenro** [comment]
i enabled new items, and then looted the conservatory pods which all have at least 5+ items.  no lag or weirdness

if turning it off fixes it for you then i'd leave it off for the moment and see if it also fixes it for others.  if it does then there might be something weird in the code that only impacts some people.

if possible could i get you to export the profile you are using and send that to me?  if its a seeting somewhere then thats the easiest way for me to have the exact same config as you

**Comment by ImSundee** [comment]
Can confirm this issue and also quantities not updating since the update.  Have to reload to refresh the inventory.

Mines actually just stopped loading at all while getting images of this:

Debug message as follows:

Date: 2021-08-13 13:04:44
ID: 14
Error occured in: Global
Count: 1
Message: ...ace\AddOns\ArkInventoryConfig\ArkInventoryConfig.lua line 9436:
   attempt to call a nil value
Debug:
   [string "=(tail call)"]: ?
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:9436: ConfigInternalLDBMountsUpdate()
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:9423: ?()
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:3390: ConfigInternal()
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:88: ConfigRefresh()
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:94: ?()
   [string "@ElvUI\Libraries\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua"]:119:
      ...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:119
   [string "=[C]"]: ?
   [string "@ElvUI\Libraries\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua"]:29:
      ...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:25
   [string "@ElvUI\Libraries\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua"]:64: SendMessage()
   [string "@ArkInventoryConfig\ArkInventoryConfig.lua"]:9496: in main chunk
   [string "=[C]"]: ?
   [string "=[C]"]: LoadAddOn()
   [string "@ArkInventory\ArkInventory.lua"]:11609: LoadAddOn()
   [string "@ArkInventory\ArkInventoryConfig.lua"]:19: Frame_Config_Show()
   [string "@ArkInventory\ArkInventoryConfig.lua"]:58:
      ArkInventory\ArkInventoryConfig.lua:57
   [string "=[C]"]: ?
   [string "@AddOnSkins\Libs\Ace3\AceConfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua"]:51:
      ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:49
   [string "@AddOnSkins\Libs\Ace3\AceConfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua"]:847:
      ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:668
   [string "=[C]"]: ?
   [string "@AddOnSkins\Libs\Ace3\AceGUI-3.0\AceGUI-3.0.lua"]:72:
      ...ddOns\AddOnSkins\Libs\Ace3\AceGUI-3.0\AceGUI-3.0.lua:70
   [string "@AddOnSkins\Libs\Ace3\AceGUI-3.0\AceGUI-3.0.lua"]:306: Fire()
   [string "@AddOnSkins\Libs\Ace3\AceGUI-3.0\widgets\AceGUIWidget-Button.lua"]:22:
      ...Libs\Ace3\AceGUI-3.0\widgets\AceGUIWidget-Button.lua:19

**Comment by arkayenro** [comment]
@ImSundee  line 2730 is blank?  and theres only 11522 lines in that file so im not sure how it got to 11609.

what version are you using?

**Comment by ImSundee** [comment]
@arkayenro 3.09.55-BurningCrusade latest from Curse

**Comment by ImSundee** [comment]
Okay full reinstall of Arc removed the loading issue. And I can't seem to reproduce the new items currently - will do some farming and update.

**Comment by ImSundee** [comment]
This morning issue is back again:

![image](https://user-images.githubusercontent.com/39481462/129440789-f59efd86-664e-41a8-9220-4e9df79f61d6.png)

You can see the new items in bag glowing, but not actually showing items. Only way to make them show is reload the UI, no errors in bugsack too.

**Comment by arkayenro** [comment]
could you try [r1145-alpha](https://www.curseforge.com/wow/addons/ark-inventory/files/3425036) and let me know if its fixed or not

**Comment by arkayenro** [comment]
actually could you try alpha 3 / [r1146-alpha](https://www.curseforge.com/wow/addons/ark-inventory/files/3425220) and let me know if its fixed or not

**Comment by DanSheps** [comment]
> i know the issue exists, but i cannot reproduce it so its a giant pain - i really wish i did have the issue

What do you need, I can reproduce it

**Comment by arkayenro** [comment]
@DanSheps try [alpha 5](https://www.curseforge.com/wow/addons/ark-inventory/files/3425820) and see if it goes away

youre looking for an item that constantly retries in the debug output

**Comment by DanSheps** [comment]
I believe it did.  I will play tonight and will let you know if I see it happen again.

**Comment by DanSheps** [comment]
@arkayenro Looks like it is still an issue.  I don't have the debug output yet, but I will enable it now.

**Comment by arkayenro** [comment]
make sure youre using [alpha 7](https://www.curseforge.com/wow/addons/ark-inventory/files/3426167)

**Comment by arkayenro** [comment]
also, try clicking on the switch character icon, then delete all backpack data.

if its doing it in the bank as well then delete that data too.

**Comment by ImSundee** [comment]
After updating to Alpha7 and clearing bags data on 2 chars, and 2 separate accounts. I'm still seeing the error, is there any error checking you'd like me do?

**Comment by arkayenro** [comment]
go into your profile an make sure that backpack data is being saved

if i turn that off i can duplicate the issue.  until i can figure out why its doing that should be a workaround

**Comment by ImSundee** [comment]
Is this what you mean?

![image](https://user-images.githubusercontent.com/39481462/129639975-d620b5b3-21ec-4f06-b6d3-d6d6cb07353a.png)
 
 
 I've always been running these settings.

**Comment by arkayenro** [comment]
yes, turning that on helps as a workaround.  i've found the actual issue and i think its fixed, just waiting for curse to update

**Comment by arkayenro** [comment]
ok, hopefully [alpha 8](https://www.curseforge.com/wow/addons/ark-inventory/files/3428782) fixes it

**Comment by ImSundee** [comment]
Have installed will test tonight and let you know.

**Comment by arkayenro** [comment]
check [alpha 10](https://www.curseforge.com/wow/addons/ark-inventory/files/3430371), that should fix the last of the issues
